modules/op-managing-approval-task-with-cli.adoc

```diff
@@ -0,0 +1,60 @@
+// This module is included in the following assemblies:
+// * create/using-manual-approval.adoc
+
+:_mod-docs-content-type: CONCEPT
```

Comment:
This section is already covered under 
"[Approving a manual approval task by using the command line"](https://98494--ocpdocs-pr.netlify.app/openshift-pipelines/latest/create/using-manual-approval.html#op-approving-task-command-line_using-manual-approval)